src/main/java/de/retest/recheck/RecheckOptions.java

```diff
@@ -224,7 +224,8 @@ public RecheckOptionsBuilder enableReportUpload() {
 		 * with {@link #addIgnore(String)}.
 		 *
 		 * @param filterName
-		 *            The filter to use for printing the differences. Default: Loads the ignore files.
+		 *            The name of the filter (including .filter extension) to use for printing the differences. Default:
```

Comment:
It is not just "printing", therefore I suggest:

```suggestion
		 *            The name of the filter (including .filter extension) to use for filtering differences. Default:
```

(Same below.)